app/routes/join.tsx

```diff
@@ -100,6 +100,7 @@ export default function Join() {
                 ref={emailRef}
                 id="email"
                 required
+                // eslint-disable-next-line jsx-a11y/no-autofocus
                 autoFocus={true}
```

Comment:
These feel like they're an OK exception to the rule as this form is the sole purpose of the page